app/javascript/apps/Projects/index.tsx

```diff
@@ -26,17 +42,24 @@ export const ProjectsIndex: React.FC<ProjectsProps> = ({
             Team: {params.tags[0]}
           </Sans>
         )}
-        {unreleasedProjects && (
-          <Box pb={4}>
-            <Sans size="6">Out of sync</Sans>
-            <ProjectSummaryGrid projects={unreleasedProjects} />
-          </Box>
-        )}
-        {releasedProjects && (
-          <Box pb={4}>
-            <Sans size="6">Up to date</Sans>
-            <ProjectSummaryGrid projects={releasedProjects} />
-          </Box>
+        {ProjectsViewSwitcher()}
```

Comment:
I _think_ you can treat this as a component! 🎉 

```suggestion
        <ProjectsViewSwitcher />
```